CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed 🐞
 
+-   Buildings are flattened when delta is active. 
```

Comment:
```suggestion
-   Buildings are flattened when delta is active #824.
```

Reply:
We might want to use full links as these are not converted to a link in the document itself.

```suggestion
-   Buildings are flattened when delta is active [#824](https://github.com/MaibornWolff/codecharta/issues/824).
```

Reply:
I will try it out :)